bioimageio/spec/v0_3/raw_nodes.py

```diff
@@ -23,14 +23,16 @@
 # such that we only need to support the most up-to-date version.
 FormatVersion = Literal["0.3.0", "0.3.1", "0.3.2"]  # newest format needs to be last (used in spec.__init__.py)
```

Comment:
Shouldn't "0.1.0" be in here?

Reply:
no, not in the v0_3 module... 
the CI should select the correct version module (this is sitll a todo), but otherwise the version modules are somewhat independent of each other.

Reply:
(with 'auto-convert' 0.1.0 might be valid as it can be ported to 0.3.x in some cases... )